PD-641 Create Call-Outs Library Sketch File

audroue opened the pull request:
- Created and added Call-Out symbol library to LG Sketch

audroue committed:
PD-641 Create Call-Outs Library Sketch File

- Created and added Call-Out symbol library to LG Sketch

DesignerDave commented:
Hey Rob! Few nits but looks good overall!
- The reference page has a layer named "Reference for testin" which doesn't seem to have a purpose.
- The cover image says "Call-Out" while the file names is plural: "Call-Outs".
- I have icons showing up in the insert menu. I'm not sure if this is another instance of my computer having issues versus this being more general, but it's worth noting.

One other note is that I love the use of emojis in the menu! I think it's quite fun, and might actually be a usability improvement as well, helping designers find what they need faster. I'd love to see us do this more.

audroue committed:
PD-641 Updated with feedback

- Changed cover name from "Call-Out" to "Call-Outs"
- Added reference symbol to reference layer and removed "for testing in title"

DesignerDave reviewed (approved):
LGTM 👍
